samples/bluetooth/central_bas/README.rst

```diff
@@ -20,11 +20,9 @@ Requirements
 
 * One of the following development boards:
 
-  * |nRF9160DK|
   * |nRF5340DK|
   * |nRF52840DK|
   * |nRF52DK|
```

Comment:
What is nRF52DK  ? 
Also we should specify the boards as defined in boards/arm directory, nrf52840_pca10056 , nrf52840_pca10059, nrf9160_pca10090 This would help us for Q3 release when we support different chips in different PCBS 

By default all BLE examples should point to nRF52840_pca10056, nRF52832_pca10040

Reply:
These are substitution shortcuts to the DK names. This section expands to the following in the output : 

- nRF5340 PDK board (PCA10095) - see :ref:`ug_nrf5340`
- nRF52840 DK board (PCA10056)
- nRF52 DK board (PCA10040)

Should we remove nRF5340 DK also from all BLE samples?

Reply:
BLE and NFC examples are tested for nRF53 target.

Reply:
Just adding nRF52840 DK does not differentiate  the chips on the board.
There are nrf52811_pca10056 as well nrf52840_pca10056
Same for nrf52810_pca10040 as well nRF52832_pca10040.